### PR TITLE
Remove generated content from main

### PR DESCRIPTION
**Pull request by scholtzan**
To avoid confusion and people from editing files that will be overwritten by generation, we should remove all generated content from being checked into the repo

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2463)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/event_types_history_v1/init.sql [hunk 1]
```diff
@@ -1,4 +1,3 @@
--- Generated by ./bqetl generate events_daily
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/event_types_history_v1/init.sql:1]: These queries are actually not being generated for `telemetry`

**Comment by scholtzan** [comment]
Looking at the diff, there are some `init.sql` files being removed for experiment enrollment queries. This is fine, the `schema.yaml` files can be used instead to deploy these tables

**Review by sean-rose — APPROVED** [review]
r+wc

**Comment by sean-rose** [comment]
I'm also a bit confused why the `sql.diff` shows that the PR is causing `schema.yaml` files to no longer be generated for the following unrelated views:
- `moz-fx-data-shared-prod.hubs_derived.active_subscription_ids_live`
- `moz-fx-data-shared-prod.relay_derived.active_subscription_ids_live`
- `moz-fx-data-shared-prod.telemetry.releases_latest`

It's probably related to #4848, but why is it showing up here?

File: sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_v1/init.sql
[the remark this answers is not in the document]
Comment (scholtzan) [reply sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_v1/init.sql:1]: The schemas will be automatically generated during the deploy: https://github.com/mozilla/telemetry-airflow/blob/4050697bd2e283864a9013bd48d439eb66f6d581/dags/bqetl_artifact_deployment.py#L72

**Pull request merged**